astropy/timeseries/periodograms/lombscargle_multiband/tests/test_lombscargle_multiband.py

```diff
@@ -0,0 +1,586 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+from astropy import units as u
+from astropy.tests.helper import assert_quantity_allclose
+from astropy.time import Time, TimeDelta
+from astropy.timeseries import TimeSeries
+from astropy.timeseries.periodograms.lombscargle_multiband import LombScargleMultiband
+
+ALL_METHODS = LombScargleMultiband.available_methods
+NORMALIZATIONS = ["standard", "psd", "log", "model"]
+
+
+@pytest.fixture
```

Comment:
I *think* if you put the pytest fixtures in a `conftest.py` file in this directory, you don't need the empty `__init__` file.

Reply:
We usually always leave the ``__init__.py`` files in this package otherwise it messes with package collection by setuptools.